tools/traffic/config/config.go

```diff
@@ -106,6 +106,9 @@ func NewConfig(ctx *cli.Context) (*Config, error) {
 			EigenDAServiceManager: retrieverConfig.EigenDAServiceManagerAddr,
 			SignerPrivateKey:      ctx.String(SignerPrivateKeyFlag.Name),
 			CustomQuorums:         customQuorumsUint8,
+
+			MetricsBlacklist:      ctx.StringSlice(MetricsBlacklistFlag.Name),
```

Comment:
why do we need both flags? 
could we just use `MetricsFuzzyBlacklist`? not sure what use cases there are where we want exact matching vs. fuzzy matching (fuzzy matching can cover all use cases we need for exact matching)

Reply:
Fuzzy matching is needed to handle metric labels that contain DA node IDs. Exact matching is needed to disable metrics where one metric is a substring of another. For example, there are two metrics labeled `read` and `read_success`. Without exact matching, we couldn't disable the `read` metric without also disabling the `read_success` metric.

The only straight forward way to avoid having two lists would be to interpret each entry as a regex. In your opinion, would you prefer to have a single list of regex blacklist arguments, or the current pattern?

Reply:
I see. That makes sense. I think the current pattern is simpler in that case